README.md

```diff
@@ -67,6 +69,11 @@ Now you should be able to go to http://localhost:4200/ and login with the [user
 # Coding Standards
 
 ## Formatting
+
+### Frontend
+We are using tslint to lint our code. Install it as per the [instructions above](#frontend-1), and then run `tslint --project .` in the `frontend` directory to run the linter and show you any errors that need to be fixed. If you are using VS Code, consider installing [the extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin).
```

Comment:
May want to mention -c /path/to/tsconfig.json too. My editor needed a hint finding it